insights/combiners/ps.py

```diff
@@ -0,0 +1,300 @@
+"""
+PS
+==
+
+This combiner provides information about running processes based on the ``ps`` command.
+More specifically this consolidates data from
+:class:`insights.parsers.ps.PsEo`,
+:class:`insights.parsers.ps.PsAuxcww`,
+:class:`insights.parsers.ps.PsEf`,
+:class:`insights.parsers.ps.PsAux`,
+:class:`insights.parsers.ps.PsAuxww` and
+:class:`insights.parsers.ps.PsAlxwww` parsers (in that specific order).
+
+Note:
+    The final dataset can vary depending on availability of the parsers
+    for a given ``ExecutionContext`` and added filters. Please see
+    :py:func:`insights.core.filters.add_filter` for the information on filtering.
+
+
+Examples:
```

Comment:
@vdymna please add tests for the examples.  See this [example](https://github.com/RedHatInsights/insights-core/blob/19a1e34dd2178906eaecdb4864e13e3cb20e8025/insights/combiners/tests/test_cloud_provider.py#L655) in another combiner.